Mixpanel/EventListener/ControllerListener.php

```diff
@@ -99,7 +101,7 @@ private function updateDataAnnotation(Annotation\Annotation $annotation, Request
         }
 
         if ($value instanceof Annotation\Expression) {
-            $element = $this->expressionLanguage->evaluate($value->expression, $request->attributes->all());
+            $element = $this->expressionLanguage->evaluate($value->expression, array_merge($request->attributes->all(), ['request' => $request]));
```

Comment:
You should take example on how Symfony used expression language for its routing there: 
https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Routing/Matcher/Dumper/CompiledUrlMatcherDumper.php#L428-L433